rpc/client.test.ts

```diff
@@ -10,12 +10,13 @@ import {
   ProviderHandlerError,
   ProviderListener,
   ProviderSendError,
+  proxyProvider,
 } from "./mod.ts"
 
 Deno.test({
   name: "RPC Client",
   async fn(t) {
-    const client = await T.polkadot.client
+    const client = new Client(proxyProvider, discoveryValue)
```

Comment:
Can't this import the raw client?

Reply:
It could, but this test is meant to cover client instantiation, discarding, etc. –– hence, best to do it manually with the discovery value